lib/dynflow/action/with_polling_sub_plans.rb

```diff
@@ -0,0 +1,72 @@
+module Dynflow
+  module Action::WithPollingSubPlans
+
+    REFRESH_INTERVAL = 10
+    Poll = Algebrick.atom
+
+    def run(event = nil)
+      case event
+      when Poll
+        poll
+      else
+        super(event)
+      end
+    end
+
+    def poll
+      recalculate_counts
+      try_to_finish || suspend_and_ping
+    end
+
+    def wait_for_sub_plans(sub_plans)
+      increase_counts(sub_plans.count, 0)
+      if is_a?(::Dynflow::Action::WithBulkSubPlans)
```

Comment:
We don't really need it, but it makes sense to start polling after all the sub plans were spawned.